linchpin/shell/__init__.py

```diff
@@ -208,8 +208,11 @@ def init(ctx):
               cls=MutuallyExclusiveOption, mutually_exclusive=["run_id"])
 @click.option('-if', '--inventory-format', default="cfg",
               help="Inventory format can be cfg or json")
+@click.option('-rhof', '--run-hooks-on-failure', is_flag=True, default=None)
+@click.option('-nh', '--no-hooks', is_flag=True, default=None)
 @pass_context
-def up(ctx, targets, run_id, tx_id, inventory_format):
+def up(ctx, targets, run_id, tx_id, inventory_format,
+       run_hooks_on_failure, no_hooks):
```

Comment:
Can this flag not be obtained from the `ctx.get_evar()` or `ctx.get_cfg()` method? Why does this need to be passed?

Reply:
@samvarankashyap  Actually, no they would not have to use what's in the config files. My point was exactly that you can modify the ctx to set the configs in the up/down methods which would use the flags from the cli. In turn, it would override the ones from linchpin.conf/linchpin.constants.